src/stories/accidenz_commons.stories.js

```diff
@@ -9,4 +9,186 @@ export const demo = () => `
   <p class="accidenz-commons">
     The quick brown fox jumps over the lazy dog.
   </p>
+
 `
+
+export const UpperCaseLetters = () => {
+    return `
+        <style>
+            .size {
+            font-size: 24pt;
+            font-family: "Accidenz Commons";
+
+            border: 3px solid rgb(85, 77, 77);
+            color: rgb(85, 77, 77);
+            padding: 25px;
+            }
+
+            .fontEg {
+            font-family: "Accidenz Commons";
+            font-size: 20pt;
+            }
+
+            .example-header {
+            margin-top: 20px;
+            }
+            th:hover {
+            background-color: rgb(224, 214, 214);
+            }
+        </style>
+        <table>
+                <tr>
+                    <th class="size">A</th>
+                    <th class="size">B</th>
+                    <th class="size">C</th>
+                    <th class="size">D</th>
+                    <th class="size">E</th>
+                    <th class="size">F</th>
+                    <th class="size">G</th>
+                </tr>
+            <tr>
+                <th class="size">H</th>
+                <th class="size">I</th>
+                <th class="size">J</th>
+                <th class="size">K</th>
+                <th class="size">L</th>
+                <th class="size">M</th>
+                <th class="size">N</th>
+            </tr>
+            <tr>
+                <th class="size">O</th>
+                <th class="size">P</th>
+                <th class="size">Q</th>
+                <th class="size">R</th>
+                <th class="size">S</th>
+                <th class="size">T</th>
+                <th class="size">U</th>
+            </tr>
+            <tr>
+                <th class="size">V</th>
+                <th class="size">W</th>
+                <th class="size">X</th>
+                <th class="size">Y</th>
+                <th class="size">Z</th>
+            </tr>
+        </table>
+
+        <h3 class="example-header">Example</h3>
+        <p class="fontEg">THE QUICK BROWN FOX JUMPS OVER LAZY DOG</p>
+    `
+}
+
+export const LowerCaseLetters = () => {
+    return `
+        <style>
+            .size {
+            font-size: 24pt;
+            font-family: "Accidenz Commons";
+
+            border: 3px solid rgb(85, 77, 77);
+            color: rgb(85, 77, 77);
+            padding: 25px;
+            }
+
+            .fontEg {
+            font-family: "Accidenz Commons";
+            font-size: 20pt;
+            }
```

Comment:
CSS should be properly indented.
```suggestion
            .font-eg {
              font-family: "Accidenz Commons";
              font-size: 20pt;
            }
```

Reply:
Okay thank you very much will update the following